content/bdk-cli/installation.md

```diff
@@ -34,59 +34,38 @@ Once Cargo is installed, you can proceed to install the interactive `bdk-cli` to
 the GitHub repository, by running:
 
 ```bash
-cargo install --git https://github.com/bitcoindevkit/bdk-cli --features=esplora
+cargo install --git https://github.com/bitcoindevkit/bdk-cli --features="compiler"
```

Comment:
Sorry but I think this line should also include `esplora` since it's mentioned on the "Interface" page. Also the quotes aren't needed.
```suggestion
cargo install --git https://github.com/bitcoindevkit/bdk-cli --features=esplora,compiler
```

Reply:
:P Not a problem at all!